pkgs/applications/system/resources/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, appstream-glib
+, cargo
+, desktop-file-utils
+, meson
+, ninja
+, pkg-config
+, rustPlatform
+, rustc
+, wrapGAppsHook4
+, glib
+, gtk4
+, libadwaita
+}:
+
+stdenv.mkDerivation rec {
+  pname = "resources";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "nokyan";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-OVz1vsmOtH/5sEuyl2BfDqG2/9D1HGtHA0FtPntKQT0=";
+  };
```

nit: consider using [`finalAttrs`](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes) (you'll need to replace more than the suggestion below)

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "resources";
  version = "1.2.1";

  src = fetchFromGitHub {
    owner = "nokyan";
    repo = finalAttrs.pname;
    rev = "refs/tags/v${finalAttrs.version}";
    hash = "sha256-OVz1vsmOtH/5sEuyl2BfDqG2/9D1HGtHA0FtPntKQT0=";
  };
```